src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -600,9 +601,10 @@ protected void setBelowMaxDepthTransactionLimit(int maxAnalyzedTransactions) {
     }
 
     public interface Defaults {
-        //API
+		//API
         int API_PORT = 14265;
         String API_HOST = "localhost";
+        String[] REMOTE_LIMIT_API = new String[] {"addNeighbors", "getNeighbors", "removeNeighbors"};
```

Comment:
I think we should also add `attachToTangle` & `interruptAttachingToTangle` to the default list.

Reply:
Change to `List<String> REMOTE_LIMIT_API`

In `IotaUtils` you can add the following method:
```
 public static <T> List<T> createImmutableList(T... a) {
        return Collections.unmodifiableList(Arrays.asList(a));
    }
```

Note that once we upgrade Java we can start using `List.of()` instead of doing this ugly thing.
See:
https://docs.oracle.com/javase/9/docs/api/java/util/List.html

Reply:
@alon-e isnt attachToTangle used in a lot of tutorials? That will probably confuse people if its disabled by default

Reply:
Since this is also a product decision I am tagging @jakubcech.
I want to remind everyone that since changing the defaults breaks backwards compatibility, perhaps it shouldn't be released together with the changes we made for configurations in #724.

Reply:
`attachToTangle` & `interruptAttachingToTangle` should be in the list. We should update our docs if it's broken by this change somewhere.